charts/rasa-x/Chart.yaml

```
@@ -1,7 +1,7 @@
 ---
 apiVersion: v2
 
-version: "1.11.0"
+version: "1.11.1"
```

```suggestion
version: "1.11.0"
```
We add a new feature, not a fix :D